transfer/src/main/java/com/kin/ecosystem/transfer/AccountInfoService.java

```diff
@@ -0,0 +1,167 @@
+package com.kin.ecosystem.transfer;
+
+import android.app.IntentService;
+import android.app.Notification;
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.ComponentName;
+import android.content.Intent;
+import android.content.pm.ResolveInfo;
+import android.graphics.Color;
+import android.net.Uri;
+import android.os.Build;
+import android.support.annotation.Nullable;
+import android.support.annotation.RequiresApi;
+import android.support.v4.app.NotificationCompat;
+import android.support.v4.content.FileProvider;
+import android.util.Log;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+
+public abstract class AccountInfoService extends IntentService {
+    public interface AccountInfoListener {
+        void onAccountInfoReady(String data);
```

Comment:
it's not clear that you are should receive publicAddress.

Reply:
for now we might need only the public address 
later on we might need more info about the account
so i prefer to write something generic and not specific

Reply:
So let's deal with it later, for now, you can just use `publicAddress` or an `AccountInfo` object  with `publicAddress` as param.
Also for something more generic, I would create an `AccountInfo` object and add params there.